deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py

```diff
@@ -120,6 +121,80 @@ def build_augmentation_pipeline(self, height=None, width=None, apply_prob=0.5):
                         )
                     )
                 )
+        if cfg.get("grayscale", False):
+            pipeline.add(sometimes(iaa.Grayscale(alpha=(0.5, 1.0))))
+
+        def get_aug_param(cfg_value):
+            if isinstance(cfg_value, dict):
+                opt = cfg_value
+            else:
+                opt = {}
+            return opt
+
+        cfg_cnt = cfg.get("contrast", {})
+        cfg_cnv = cfg.get("convolution", {})
+
+        contrast_aug = ["histeq", "clahe", "gamma", "sigmoid", "log", "linear"]
+        for aug in contrast_aug:
+            aug_val = cfg_cnt.get(aug, False)
+            cfg_cnt[aug] = aug_val
+            if aug_val:
+                cfg_cnt[aug + "ratio"] = cfg_cnt.get(aug + "ratio", 0.1)
+
+        convolution_aug = ["sharpen", "emboss", "edge"]
+        for aug in convolution_aug:
+            aug_val = cfg_cnv.get(aug, False)
+            cfg_cnv[aug] = aug_val
+            if aug_val:
+                cfg_cnv[aug + "ratio"] = cfg_cnv.get(aug + "ratio", 0.1)
```

Comment:
What do you say about the following?
```python
contrast_aug = ["histeq", "clahe", "gamma", "sigmoid", "log", "linear"]
convolution_aug = ["sharpen", "emboss", "edge"]
for aug in contrast_aug + convolution_aug:
    ...
```

Reply:
Yeah, fine

Reply:
Didn't even know that python supports + in a for statement! Crazy language. Pushed this now :)

Reply:
Yeah, lists get concatenated first ☺️

Reply:
Tests failing now though? What did I do 🤯

Reply:
fixed..

Reply:
My fault, sorry 🙊